docs/reference/modules/node.asciidoc

```diff
@@ -340,6 +340,9 @@ xpack.ml.enabled: true <1>
 ----
 <1> The `xpack.ml.enabled` setting is enabled by default.
 
+If you want the node to use {ccs}, add `remote_cluster_client` to the list of
+roles. See <<remote-node>>.
```

Comment:
I think we need some clarification on whether a node with `ml` and `remote_cluster_client` is still considered a dedicated ML node.  I think it should still be:

```suggestion
roles. See <<remote-node>>.  A node with both the `ml` and `remote_cluster_client` roles
but no others is still considered a dedicated {ml} node.
```